go/common/headers.go

```diff
@@ -26,7 +26,7 @@ type BatchHeader struct {
 	UncleHash   common.Hash    `json:"sha3Uncles"`
 	Coinbase    common.Address `json:"miner"`
 	Root        StateRoot      `json:"stateRoot"`
-	TxHash      common.Hash    `json:"transactionsRoot"` // todo - include the synthetic deposits
+	BodyHash    common.Hash    `json:"transactionsRoot"` // corresponds to block's `TxHash`. todo - include the synthetic deposits
```

Comment:
for the batch it still makes sense to call it TxHash.
Because the batch will only contain transactions in the body